R/utilities-sensitivity-calculation.R

```diff
@@ -89,17 +89,16 @@
   pkData <- .addParameterColumns(pkData, simulationResults, parameterPath)
   pkData <- dplyr::group_by(pkData, ParameterPath, PKParameter) %>%
     dplyr::group_modify(.f = ~ .computePercentChange(.)) %>%
-    dplyr::ungroup()
+    dplyr::ungroup() %>%
+    dplyr::select(
+      "OutputPath",
+      dplyr::starts_with("Parameter"),
+      dplyr::starts_with("PK"),
+      Unit,
+      dplyr::everything(),
```

Comment:
why use conditionnals and then `everything()` and a `-` right after ? Is it for reordering the columns ? Is it necessary to reorganize them ? I know this is not really part of this PR but I would clean this or add a comment explaining why.

At first sight, only saying to the select fonction "all variable but `individualId`" should be enough. Translated to dplyr language it would be:
```r
dplyr::select(-any_of("IndividualId"))
```

And then use `dplyr::relocate()` to reorganize columns:
```r
dplyr::relocate(
 "OutputPath",
  dplyr::starts_with("Parameter"),
  dplyr::starts_with("PK"),
  Unit
)

Reply:
I guess the intention was to reorder the columns, but I agree that this looks a bit convoluted. Yes, also I try not to touch code which is not related to the PR.

Reply:
To be honest, I ould leave it like that:

``
dplyr::select(
      "OutputPath",
      dplyr::starts_with("Parameter"),
      dplyr::starts_with("PK"),
      Unit,
      dplyr::everything(),
      -dplyr::any_of("IndividualId")
    )
``

Use `any_of()` seems reasonable and is recommended. But reordering columns using `select()`  is an acceptable approach. For me the benefit of not using an additional function `dplyr::relocate` overweights.

Reply:
Question is how are `dplyr::select()` performance on big dataframes compared to `dplyr::relocate()` ?